nipype/interfaces/ants/segmentation.py

```diff
@@ -1129,11 +1129,28 @@ def _format_arg(self, opt, spec, val):
             retval = ''
             if not isdefined(self.inputs.out_label_fusion):
                 retval = '-o {0}'.format(self.inputs.out_intensity_fusion_name_format)
+        elif opt == 'atlas_image':
```

Comment:
I think so.  My goal for this patch was to make the minimal number of changes necessary return AntsJointFusion to a working state.  The previous command line generated had problems with formatting the command lines due to the way that lists of strings were being created.  i.e. "-g %s" for [u'/file_a', u'/file_b'] was being presented as "-g [u/file_a, u/file_b]" on the command line.